libfwupdplugin/tests/bios-attrs/lenovo-p620-6.3/thinklmi/attributes/AutomaticBootSequence/display_name

```diff
@@ -0,0 +1 @@
+AutomaticBootSequence
```

Comment:
can you symlink any of the existing tree contents?

Reply:
Certainly can, but that's a lot of manual work and intend to drop the old folder eventually.

Reply:
Ohh, I thought you could just point `fdupes` at it...

Reply:
6.6M source to 4.9M source by making this change!